domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/PaymentStatus.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Arrays;
+
+@Getter
+@AllArgsConstructor
+public enum PaymentStatus {
+    INITIATED("Initiated"),
+    SUCCESS("Success"),
+    FAILED("Failed"),
+    PENDING("Pending"),
+    DECLINED("Declined");
```

Comment:
Heya - are we confident we have got an exhaustive list of statuses here?

Reply:
yep, I've taken that from the ccpay codebase itself :)